api/data_refinery_api/views.py

```diff
@@ -46,7 +46,7 @@
 from rest_framework.settings import api_settings
 from rest_framework.views import APIView
 
-from data_refinery_api.serializers import (
+from .serializers import (
```

Comment:
Why are you changing this to use relative imports?

Reply:
Because that way the IDE can locate all those types and provide code competition and type checking. Does that have any other drawbacks?

Reply:
Why is that necessary? I don't understand why that'd make a difference.

Pep8 has this to say:
```
Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):
...
However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:
...

Standard library code should avoid complex package layouts and always use absolute imports.

Implicit relative imports should never be used and have been removed in Python 3.
```

Reply:
ok I'll try to find another way. I saw relative imports in other parts of the code, we might want to update those as well.